debugger/src/components/Debugger/Signals/Signal/Action/index.js

```diff
@@ -42,60 +43,90 @@ function renderCode (error) {
+        Prism.highlightElement(this.errorElement)
+        this.isHighlighted = true
+      }
+    })
+  }
+  render () {
+    const {action, faded, execution, children, onMutationClick, executed} = this.props
+
+    const error = execution && execution.error
+    const titleClassname = classnames({
+      'action-actionErrorHeader': error,
+      'action-actionHeader': !error,
+      'action-faded': faded
+    })
+    return (
       <div
-        className={titleClassname}
-        onClick={() => onActionClick(action)}>
-        {error ? <i className='icon icon-warning' /> : null}
-        {action.isAsync ? <i className='icon icon-asyncAction' /> : null}
-        {renderActionTitle(action)}
-      </div>
-      {error ? (
-        <div className='action-error'>
-          <div className='action-actionInput'>
-            <div className='action-inputLabel'>props:</div>
-            <div className='action-inputValue'><Inspector value={execution.payload} /></div>
-          </div>
-          <strong>{error.name}</strong> : {error.message}
-          <pre data-line={getLineNumber(error)}>
-            <code className='language-javascript' dangerouslySetInnerHTML={{__html: renderCode(error)}} />
-          </pre>
+        className={error ? 'action action-actionError' : 'action'}
+        onClick={(event) => event.stopPropagation()}
+      >
+        <div className={titleClassname}>
+          {error ? <i className='icon icon-warning' /> : null}
+          {action.isAsync ? <i className='icon icon-asyncAction' /> : null}
+          {renderActionTitle(action)}
         </div>
-      ) : null}
-      {!error && execution ? (
-        <div>
-          <div className={faded ? 'action-faded' : null}>
+        {error ? (
+          <div className='action-error'>
             <div className='action-actionInput'>
               <div className='action-inputLabel'>props:</div>
               <div className='action-inputValue'><Inspector value={execution.payload} /></div>
             </div>
-            <div className='action-mutations'>
-              {execution.data.filter(data => data.type === 'mutation').map((mutation, index) => <Mutation mutation={mutation} key={index} onMutationClick={onMutationClick} />)}
-            </div>
-            <div className='action-services'>
-              {execution.data.filter(data => data.type !== 'mutation').map((service, index) => <Service service={service} key={index} />)}
+            <div className='action-error-message'>
+              <strong>{error.name}:</strong> <Inspector value={error.message} />
             </div>
+            <pre data-line={getLineNumber(error)}>
+              <code
+                ref={(node) => { this.errorElement = node }}
+                className='language-javascript'
+                dangerouslySetInnerHTML={{__html: renderCode(error)}} />
+            </pre>
             {executed}
-            {execution.output ? (
+          </div>
+        ) : null}
```

Comment:
Fixed... though not sure if this is good idea. Cause very often you check a boolean, where you have to do `someBool ? (<jsx/>) : null`. Where `someBool && (<jsx />)` would return `false`, not `null` or `undefined`. Rather be consistent than giving the overhead of figuring out if it is actually valid to shorten it. hm hm... changed it now, but yeah. We should probably decide on this